pkg/envoy/server.go

```diff
@@ -168,6 +168,7 @@ func StartXDSServer(stateDir string) *XDSServer {
 							"config": {Kind: &structpb.Value_StructValue{StructValue: &structpb.Struct{Fields: map[string]*structpb.Value{}}}},
 						}}}},
 					}}}},
+					"stream_idle_timeout", {Kind: &structpb.Value_StructValue{StructValue: &structpb.Struct{Fields: map[string]*structpb.Value{}}}},
```

Comment:
Typo: `"stream_idle_timeout",` -> `"stream_idle_timeout":`. (use colon, not comma)

Reply:
Fixed... sorry for the churn!